client/components/Help/HelpPortalModal/HelpPortalModal.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect, useCallback, FC } from 'react'
+import { Modal, ModalHeader, ModalBody } from 'reactstrap'
+
+type Props = {
+  el: HTMLElement | null
+}
+
+const HelpPortalModal: FC<Props> = ({ el = null }) => {
```

Comment:
I think component interfaces should not be specially shaped for Storybook.
`el` props will not be needed.
```suggestion
const HelpPortalModal: FC<{}> = () => {
```

Reply:
done.